robot_ws/.rosinstall

```diff
@@ -0,0 +1 @@
+- git: {local-name: src/deps/lex-ros2, uri: "https://github.com/aws-robotics/lex-ros2.git", version: "master"}
```

Comment:
We'll want to document somewhere that this should be removed once https://github.com/aws-robotics/lex-ros2/pull/7 has been released to apt

Reply:
Have noted it in my task list.